### ISSUE TITLE
Contextual menu in repository

### ISSUE THREAD

**Issue by alKODIque** [jurialmunkey/plugin.video.themoviedb.helper#1201]
### Describe the feature that you'd like

I can no longer find the Context Menu extension for TMDb-Helper in the v3.4 repository

### Why is this feature needed? What problem does it solve?

I was wondering if this was normal and intended, or a simple oversight ;)

### Checklist

- [X] I have searched the issues and this feature has not been requested.
- [X] I have updated TMDbHelper to the most recent version.

**Comment by booyakafoo** [comment]
I was looking for that also. Can confirm it doesn't show up in the repository with Kodi v21.

**Comment by jurialmunkey** [comment]
I haven't touched the context menu in ages. Not even sure it still works correctly. All the options are already available from the main context menu on tmdbhelper items anyway.

Context addon only for non tmdbhelper items and does a lot of guess work to match items. You can install manually from the github code button zip but use at your own risk.

**Comment by booyakafoo** [comment]
Yeah. I'll install manually and try it out.  Its useful to have when an autoplay player defaults to a weird video file when using "fake" library .strm files.

**Comment by booyakafoo** [comment]
Installed the [context.themoviedb.helper-matrix.zip](https://github.com/jurialmunkey/context.themoviedb.helper/archive/refs/heads/matrix.zip) from the [context.themoviedb.helper repository](https://github.com/jurialmunkey/context.themoviedb.helper) manually and it is working well for me so far.  Thanks.

**Comment by alKODIque** [comment]
@booyakafoo 
That's what I did 👍
The context menu is very useful, since we can use it from other addons.  We can add media to local  library, viewTrakt options, browse the lists, etc
If we dont have this context menu, we can of course have all these options without it, but only from TMDb-Helper.
@Jurialmunkey:
That's why it's very useful despite the fact that you haven't worked on this script for a long time :P !! It's still functional and has no bugs ;)

**Comment by jurialmunkey** [comment]
It definitely has bugs. One person opened a bug issue about using it just last week complaining they were getting an error using it. That's why it isn't in my repo.

**Comment by derrgoo** [comment]
I use it sometimes and works fine for me. However as of late it is invisible on context menu but the space is there.
